frontend/packages/console-shared/src/components/lists/DetailPropertyListItem.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { ListItem } from '@patternfly/react-core';
+
+type Props = {
+  title?: string;
```

Comment:
\+ children?

Reply:
`children` is already included in React.FC via React.PropsWithChildren type (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L518)